.circleci/config.yml

```diff
@@ -64,6 +64,10 @@ commands:
           name: Generate dev and prod archives
           command: |
             ./scripts/archive.sh << parameters.artifact_dir >> $BUILD_VERSION
+      - run:
+          name: Check expected artifacts exist
+          command: |
+            ./scripts/archive-test.sh << parameters.artifact_dir >> $BUILD_VERSION
```

Comment:
📣 We will now fail CI if an expected executable isn't found! IIRC this has caught us off guard before with `latest` releases so these tests were improved too in this PR.